src/Fhir.Anonymizer.Shared.Core/Constants.cs

```diff
@@ -30,5 +28,7 @@ internal static class Constants
 
         internal const string GeneralResourceType = "Resource";
         internal const string GeneralDomainResourceType = "DomainResource";
+
+        internal static Dictionary<string, string> allowedVersion = new Dictionary<string, string> { { "3", "Stu3" }, { "4", "R4" } };
```

Comment:
For now it is safe. But we should keep in mind that there will be a R5 update in the future